lib/shortcode/bbpress.php

```diff
@@ -0,0 +1,259 @@
+<?php
+
+/**
+ * Shortcode Template File for BBPress
+ */
+
+/**
+ * Commonly-used variables stored here for better management.
+ */
+
+function sandwich_bbpress_display_view () {
+	$output['popular'] = __( 'Popular', 'pbsandwich' );
+	$output['no-replies'] = __( 'No Replies', 'pbsandwich' );
+	return $output;
+}
+
+/**
+ * Encodes the list of products into an array variable.
+ * Choose between array, comma-separated string or slug.
+ */
+
+function sandwich_bbpress_posttype_list ( $type = "forum", $id = "false" ) {
+ 	$args = array(
+ 		'post_type' => $type,
+ 		'posts_per_page' => '-1'
+ 		);
+ 	$loop = new WP_Query( $args );
+ 	if ( $loop->have_posts() ) {
+		$output[0] = __( 'Make your selection', 'pbsandwich' );
```

Comment:
Standardize all `0` *Select* dropdowns as `sprintf( '— %s —', __( 'Select', 'pbsandwich' ) )`

Please **reuse** translation strings so that our translation file is smaller